### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by Syul968**
Casing, tests, ids...

### CODE REVIEW

**Comment by RComonfort** [comment]
Please fix conflicts in order to review :3

File: server/middleware/validations/tutoring-validation.js [hunk 1]
```diff
@@ -87,19 +87,19 @@ const validateTutoring = async (tutoring, required, td, st, et) => {
...
-        return ErrorFactory.buildError(Errors.INVALID_ID, 'userID', tutoring.userID);
+    if(tutoring.topicId && !ObjectId.isValid(tutoring.topicId))
+        return ErrorFactory.buildError(Errors.INVALID_ID, 'topicId', tutoring.topicId);
+    if(tutoring.userId && !ObjectId.isValid(tutoring.userId))
+        return ErrorFactory.buildError(Errors.INVALID_ID, 'userId', tutoring.userId);
     if(tutoring.tutorId && !ObjectId.isValid(tutoring.tutorId))
-        return ErrorFactory.buildError(Errors.INVALID_ID, 'tutorId', tutoring.tutorId);
-    if(tutoring.topicID){
-        let topic = await Topic.findById(tutoring.topicID);
+        return ErrorFactory.buildError(Errors.INVALId_ID, 'tutorId', tutoring.tutorId);
```
Comment (AngelRRuizM) [inline server/middleware/validations/tutoring-validation.js]: Is this ok?

Comment (Syul968) [reply server/middleware/validations/tutoring-validation.js]: No.

**Review by AngelRRuizM — APPROVED** [review]

File: server/models/user.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ const userSchema = new mongoose.Schema({
                     }
                 }]
             },
-            workExperience: {
+            workExperiences: {
```
Comment (RComonfort) [inline server/models/user.js]: Where is the workExperience model?

**Review by RComonfort — CHANGES_REQUESTED** [review]

**Comment by RComonfort** [comment]
Also, need to solve conflicts still

File: server/models/user.js (hunk 2 above)
Comment (Syul968) [reply server/models/user.js]: In `sprint3` 😃

**Review by RComonfort — APPROVED** [review]

**Comment by RComonfort** [comment]
Now thats more like it

**Pull request merged**